src/pandas_profiling/model/expectation_algorithms.py

```diff
@@ -0,0 +1,70 @@
+def generic_expectations(name, summary, batch, *args):
+    batch.expect_column_to_exist(name)
+
+    if summary["n_missing"] == 0:
+        batch.expect_column_values_to_not_be_null(name)

```

Comment:
Just a heads-up that it is possible to use the `mostly` parameter here, so for instance, if the % missing was less than 10%, you could use `batch.expect_column_values_to_not_be_null(name, mostly=0.9)`. I would say it's your call as to whether to use this, and what the threshold should be if you do.